src/main/java/com/commercetools/sync/inventories/utils/InventoryUpdateActionUtils.java

```diff
@@ -43,7 +42,7 @@ private InventoryUpdateActionUtils() {
                                                                                    @Nonnull final InventoryEntryDraft
                                                                                        newEntry) {
         final Long oldQuantityOnStock = oldEntry.getQuantityOnStock();
-        final Long newQuantityOnStock = newEntry.getQuantityOnStock() == null ? Long.valueOf(0L) : newEntry
+        final Long newQuantityOnStock = newEntry.getQuantityOnStock() == null ? NumberUtils.LONG_ZERO : newEntry
```

Comment:
@heshamMassoud Yes, it introduce  [slight improvement](https://github.com/commercetools/commercetools-sync-java/pull/19#discussion_r117968367)